src/PowerShell/Microsoft.WinGet.Client/Common/ErrorCode.cs

```diff
@@ -0,0 +1,16 @@
+﻿// -----------------------------------------------------------------------------
+// <copyright file="ErrorCode.cs" company="Microsoft Corporation">
+//     Copyright (c) Microsoft Corporation. Licensed under the MIT License.
+// </copyright>
+// -----------------------------------------------------------------------------
+
+namespace Microsoft.WinGet.Client.Common
+{
+    /// <summary>
+    /// Error code constants.
+    /// </summary>
+    public class ErrorCode
+    {
+        public const int FILE_NOT_FOUND = unchecked((int)0x80070002);

```

Comment:
>FILE_NOT_FOUND [](http://example.com/codeflow?start=25&length=14)

This results in a warning
warning SA1310: Field 'FILE_NOT_FOUND' should not contain an underscore

Reply:
Removed underscore to avoid warnings